Dockerfile

```diff
@@ -25,9 +25,8 @@ WORKDIR ${HOME}
 # Clean image
 RUN sudo apt-get clean && sudo rm -rf /var/lib/apt/lists/*
 
-FROM core-dependencies AS build-test
+FROM core-dependencies as build-test
 
 COPY ./ ./
-RUN cd cpp && mkdir build && cd build && cmake -DBUILD_TESTING=ON .. \
-  && make -j all && CTEST_OUTPUT_ON_FAILURE=1 make test
-RUN cd python && pip install ./ && python3 -m unittest discover ./network_interfaces/tests --verbose
+RUN cd cpp && mkdir build && cd build && cmake -DBUILD_TESTING=ON .. && make && CTEST_OUTPUT_ON_FAILURE=1 make test
+RUN cd python && pip install ./ && python3 -m unittest discover ./tests --verbose
```

Comment:
Can you name it `test` or rename the `cpp` one to `tests`. It is inconsistent otherwise

Reply:
as far as i understand `make test` is a command, it means it will run the gtests. `tests` is then the folder name, same in cpp and python

Reply:
yes my point. The folder is named `test` in `cpp` but `tests` in `python` so not the same